packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -260,7 +260,7 @@ export function getStylesDeclarations(
 
 	// The goal is to move everything to server side generated engine styles
 	// This is temporary as we absorb more and more styles into the engine.
-	const extraRules = getCSSRules( blockStyles );
+	const extraRules = getCSSRules( blockStyles, {} );
```

Comment:
Do we still need to pass in the empty object if we're already adding it as default parameter below?

Reply:
The typescript linter wouldn't stop pestering me on this one. I added it in to shut it up, but forgot to take it out. 😄